README.md

```diff
@@ -1,16 +1,73 @@
-# Title
+# JavaScript Class Extension Lab: `super`
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+1. Practice building additional functionality into subclasses
 
 ## Introduction
 
-## SWBAT 1
+In this lab we're going to create a geometry application that allows us to
+calculate different properties of multiple shape. We'll be using JavaScript's
+`class` keyword to build objects that inherit from its base object.
 
-## SWBAT 2
+## Practice Building Additional Functionality Into Subclasses
 
-## Conclusion
+1. Define a `Quadrilateral` class that accepts *four* integer parameters.
+* Define a `saturation` property set as `255` that will be inherited by subclasses.
+* Define a `perimeter()` method that calculates perimeter.
+* Define a conditional `color()` method that checks for red, green, blue, yellow, cyan, magenta, and black. It returns a `String` value of the color name (eg: "red").
+
+2. Define a `Square` subclass.
+* Define a static `name` property of `square`.
```

Comment:
static as in a static class method?

```
static name() {}
```

This is interesting, but I'm curious as to why - since the class itself is a subclass with a unique name

Reply:
You're right about this. I forgot it's a default method.